packages/common/message-source.js

```diff
@@ -68,6 +68,7 @@ class StateMachineS3MessageSource extends MessageSource {
     this.originalMessage = message;
 
     if (message.resources) {
+      // BUCKET/KEY TO DO - what bucket should this be?
```

Comment:
Still not 💯 % on this one... I'm not really sure where resources gets created, but it seems like we might not need to do anything here? Would like someone to take a look.